torchci/pages/nightlies.tsx

```diff
@@ -400,7 +200,6 @@ export default function Page() {
             }
             queryName={"nightly_jobs_red_by_platform"}
             queryParams={timeParams}
```

Comment:
The title of this panel and what it is actually querying don't match up: timeParams contains the repo parameter, which is currently set to pytorch, so the query went from looking for pytorch and domains to just pytorch.  I don't know if this is intentional

Reply:
yes just pytorch should be enough here